posthog/queries/session_recordings/session_recording_list.py

```diff
@@ -151,15 +147,10 @@ class SessionRecordingList(EventQuery):
         any(session_recordings.start_time) as start_time,
         any(session_recordings.end_time) as end_time,
         any(session_recordings.duration) as duration,
-        any(session_recordings.distinct_id) as distinct_id,
-        any(first_pageview_event.pageview_properties) as first_pageview_event_properties
+        any(session_recordings.distinct_id) as distinct_id
     FROM (
         {core_recordings_query}
     ) AS session_recordings
-    LEFT OUTER JOIN (
-        {core_single_pageview_event_query}
-    ) AS first_pageview_event
-    ON session_recordings.session_id = first_pageview_event.pageview_session_id
     JOIN (
         {person_distinct_id_query}
```

Comment:
Is this and the person_query below always necessary?

Reply:
answered by Neil in slack